recipe/meta.yaml

```diff
@@ -21,36 +22,198 @@ build:
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_saves_to_run_directory[no_wandb_core-audio_media]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_saves_to_run_directory[wandb_core-image_media]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_saves_to_run_directory[wandb_core-video_media]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_saves_to_run_directory[wandb_core-audio_media]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_with_path_traversal[no_wandb_core]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_with_path_traversal[wandb_core]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_prefixed_with_multiple_slashes[no_wandb_core-////image]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_prefixed_with_multiple_slashes[no_wandb_core-my///image]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_prefixed_with_multiple_slashes[wandb_core-////image]
+    # FAILED tests/unit_tests/test_media/test_media_logging.py::test_log_media_prefixed_with_multiple_slashes[wandb_core-my///image]
+
+    - rm tests/unit_tests/test_system_metrics/test_open_metrics.py
+    # FAILED tests/unit_tests/test_system_metrics/test_open_metrics.py::test_dcgm[no_wandb_core]
+    # FAILED tests/unit_tests/test_system_metrics/test_open_metrics.py::test_dcgm[wandb_core]
+
+    - rm tests/unit_tests/test_artifacts/test_storage.py
+    # ERROR tests/unit_tests/test_artifacts/test_storage.py::test_opener_works_across_filesystem_boundaries[no_wandb_core]
+    # ERROR tests/unit_tests/test_artifacts/test_storage.py::test_opener_works_across_filesystem_boundaries[wandb_core]
+
+    - rm tests/unit_tests/test_lib/test_hashutil.py
+    # ERROR tests/unit_tests/test_lib/test_hashutil.py::test_md5_file_hashes_on_mounted_filesystem[no_wandb_core-1kB]
+    # ERROR tests/unit_tests/test_lib/test_hashutil.py::test_md5_file_hashes_on_mounted_filesystem[no_wandb_core-50kB]
+    # ERROR tests/unit_tests/test_lib/test_hashutil.py::test_md5_file_hashes_on_mounted_filesystem[wandb_core-1kB]
+    # ERROR tests/unit_tests/test_lib/test_hashutil.py::test_md5_file_hashes_on_mounted_filesystem[wandb_core-50kB]    
+
+    # - python tools/local_wandb_server.py start &
+    - pytest tests -v
+
   requires:
+    - python
     - pip
+    - pytest
+    - pyte
+    - fastapi
+    - responses
+    - hypothesis  # ???
+    - mlflow  # ???
+    - plotly  # ???
+    - httpx
+    - uvicorn
+    - rdkit
+    - filelock
+    - IPython
+    - nbformat
+    - nbclient
+    - pytest-timeout
+    - flaky
+    - pytest-asyncio
+    - boto3
+    - jax
+    - openai
+    - tensorflow
+    - gymnasium
+    - torcheval  # for torch
+    - tensorboard
+    - google-cloud-storage
+    # - stable-baselines3
+    - tensorboardX
+    - polars
```

Comment:
An error on `win-64`:
``polars` isn't available on the main channel.`

Enable `channels` in `abs.yaml` to get dependencies from the `sfe1ed40` channel
```
channels:
  - sfe1ed40
```